lib/asciidoctor/converter/docbook5.rb

```diff
@@ -245,6 +245,13 @@ def literal node
       end
     end
 
+    begin
+      require 'asciimath'
```

Comment:
Could you change this line to:

```
require 'asciimath' unless defined? ::AsciiMath
```

That way it doesn't attempt to go down the require hole when we've already got it.